impl/pom.xml

```diff
@@ -27,6 +27,10 @@
     <artifactId>okta-jwt-verifier-impl</artifactId>
     <name>Okta JWT Verifier :: Impl</name>
 
+    <properties>
+        <jjwt.version>0.10.7</jjwt.version>
```

Comment:
use latest version, it's 0.11.x (IIRC)

Reply:
Sure, this section can be removed. The parent pom has a properties section that has a jjwt version `0.11.2` defined in it.

Reply:
Fixed.